### PR TITLE
Add Oso to "Security and PKI"

### PR DESCRIPTION
**Pull request by ssglaser**
### Free SaaS Offering Submission

Thank you for contributing to this list. This list is for **SaaS**
services that offer a **free tier** to help developers evaluate and
build something that users can later use and get support for.

The focus of this list is quite broad but we try to keep things
limited to that which infrastructure developers, like DevOps Practitioners,
would find useful.

We do not anymore accept cPanel like PHP+MySQL hosting services.

### New Submission Check List

Please ensure your submission ticks all of these boxes:

 - [ x] This is Software as a Service not self hosted
 - [ x] It has a free tier not just a free trial
 - [ x] The submission mentions what is free
 - [ x] The submission is not already present in the list

### Thank You

This list is the result of more than a thousand people contributing
to make something useful, we appreciate your efforts.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -592,6 +592,7 @@ Table of Contents
   * [onelogin.com](https://www.onelogin.com/) — Identity as a Service (IDaaS), Single Sign-On Identity Provider, Cloud SSO IdP, 3 company apps and 5 personal apps, unlimited users
   * [Operous](https://operous.dev/) — Cloud instance testing tool with a comprehensive and automated set of test-suites of best practices, performance, and security. Free tier offers 100 testing minutes, 10 Test Suites, and up to 5 instances to 1 user.
   * [opswat.com](https://www.opswat.com/) — Security Monitoring of computers, devices, applications, configurations,... Free 25 users and 30 days history users.
+  * [Oso](https://cloud-docs.osohq.com/) - Application Authorization. Authorization-as-a-service providing abstractions for building and iterating on authorization in your application. Free access to Oso Cloud with a Developer Account.
```
Comment (ripienaar) [inline README.md:595]: Do we know if they will be free also after beta?

Comment (ssglaser) [reply README.md:595]: yes, there will be a free Developer tier after beta.

Comment (ripienaar) [reply README.md:595]: Please show a link detailing that

Comment (FatGrizzly) [reply README.md:595]: seems like its no more in beta...

**Pull request closed without merging**